src/main/java/com/meta/cp4m/message/WAMessageHandler.java

```diff
@@ -257,4 +284,36 @@ private void markRead(Identifier phoneNumberId, String messageId) {
       LOGGER.error("unable to mark message as read", e);
     }
   }
+
+  private String getUrlFromID(String mediaID) throws IOException {
+     String graphAPI = "https://graph.facebook.com/v20.0/";
+   try {
+
+     Content content = Request.get(graphAPI + mediaID)
+             .setHeader("Authorization", "Bearer " + accessToken)
+             .setHeader("appsecret_proof", appSecretProof)
+             .execute().returnContent();
```

Comment:
let's use the handleResponse method instead it's suggested by the documentation to lower memory usage